src/index/index-monitor.service.ts

```diff
@@ -99,7 +100,8 @@ export class IndexMonitorService {
   }
 
   async processBlock(block: Block) {
-    this.logger.debug(`index-monitor: processing block ${block.height}`);
+    // @todo: remove comment
```

Comment:
???
Useless todo

Reply:
forgot about it, should have removed before PR :) done